.github/workflows/benchmark.yml

```
@@ -26,29 +26,29 @@ jobs:
   benchmark:
     name: TUN Throughput Benchmark
     runs-on: ubuntu-latest
-    
+
     steps:
       - name: Checkout PR branch
         uses: actions/checkout@v6
         with:
           fetch-depth: 0  # Full history for comparison
           ref: ${{ needs.check-trigger.outputs.pr_sha || github.event.pull_request.head.sha }}
```

actionlint (1.7.12)

[error] property "check-trigger" is not defined in object type {}

(expression)